spec.bs

```diff
@@ -456,9 +456,13 @@ The <dfn attribute for=HTMLFencedFrameElement>config</dfn> IDL attribute getter
 
   1. Let |urn uuid| be the given {{FencedFrameConfig}}'s [=fencedframeconfig/urn=].
 
+  1. Let |shared storage context| be the given {{FencedFrameConfig}}'s [=fencedframeconfig/
+     sharedStorageContext=].
+
   1. [=Navigate=] |element|'s [=fenced navigable container/fenced navigable=] to |urn uuid| using
      |element|'s [=Node/node document=], with [=historyHandling=] set to "<a for="history handling
-     behavior">`replace`</a>" , and [=referrerPolicy=] set to <a>"`no-referrer`"</a>.
+     behavior">`replace`</a>", [=referrerPolicy=] set to <a>"`no-referrer`"</a>, and
+     sharedStorageContext set to |shared storage context|.
```

Comment:
Since your modification to #navigate added a normal positional optional parameter, you don't need to reference its name. Generally we only add linkable/named parameters to this algorithm (and reference those names in the calling of #navigate) if the argument is directly used by another algorithm: https://docs.google.com/document/d/1yxnzjRDVmAR5CC9GcAyY448lBD0u0E98eUEMHDhx1Dw/edit?disco=AAAAeXYly54.

It's hard to say how much we're sticking with that these days, but maybe we are? In any case, I'd recommend either dropping the specific name here (since it is linked), or make the parameter name that you added _linkable_, and then link to it here.